.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,27 @@
+
+**What type of PR is this?**
+> Uncomment only one:
+>api-change
+>bug
+>cleanup
+>design
+>documentation
+>failing-test
+>feature
+
+**What this PR does / why we need it**:
+
+**Special notes for your reviewer**:
+
+**Does this PR introduce a user-facing change?**:
+
+<!--
+
+Is a release note needed ?
+If no, just write "NONE" in the release-note block below.
+If yes, enter a single sentence release note below.
```

Comment:
Should also mention to add the `release-note` label to the PR

Reply:
done